### PR TITLE
[GuestLoader]: Add autoboot for guestloader

### PR DESCRIPTION
**Pull request by in1004kyu**
1. Add CLI (command.*)
2. Add autoboot (main.c)
3. Fix guestloader's memory map(Add sp for system/user mode)

CLI is worked on RTSM. Arndale is not supproted.

### CODE REVIEW

**Comment by kesl** [comment]
CI : Test PASSed.
Refer to this link for build results: http://keslproject.iptime.org:8082/job/k-hypervisor_pull_request/131/

File: common/guest/command/command.c [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#include <command.h>
+#include <loadguest.h>
+#include <version.h>
+
+void command_help(int argc, char **argv)
```
Comment (Wonseok) [inline common/guest/command/command.c:5]: s/command_help/command_usage

Comment (Wonseok) [inline common/guest/command/command.c:5]: You do not need parameters.

File: common/guest/command/command.c [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+{
+    uart_print("load_cmd_help : help\n"
+               "help (h)        - List commands and their usage\n\n"
+               "boot (b)        - Boot guestos\n\n"
+               "banner (ba)     - Showing K-hypervisor banner\n\n"
+               "status (s)      - Showing hypervisor status\n\n");
+}
+
+void command_boot(int argc, char **argv)
```
Comment (Wonseok) [inline common/guest/command/command.c:14]: Ditto.

Comment (Wonseok) [inline common/guest/command/command.c:14]: command_boot is wrong name.

File: common/guest/command/command.c [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+#define ARM_MAX_ARG_SIZE    32
+
+void command_exec(char *line)
+{
+    int argc = 0, pos = 0, cnt = 0;
+    char *argv[ARM_MAX_ARG_SIZE];
+
+    while (line[pos] && (argc < ARM_MAX_ARG_SIZE)) {
+        if ((line[pos] == '\r') ||
+                (line[pos] == '\n')) {
```
Comment (Wonseok) [inline common/guest/command/command.c:41]: odd indentation.

File: common/guest/command/command.c [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            line[pos] = '\0';
```
Comment (Wonseok) [inline common/guest/command/command.c:42]: Use named constant.

File: common/guest/command/command.c [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        } else {
+            if (cnt == 0) {
+                argv[argc] = &line[pos];
+                argc++;
+            }
+            cnt++;
+        }
+        pos++;
+    }
+    if (argc) {
```
Comment (Wonseok) [inline common/guest/command/command.c:59]: Use case statement rather than if.

**Pull request closed without merging**